server-config.sh

```diff
@@ -19,6 +19,8 @@ apt upgrade -y
 apt install git nginx openjdk-11-jdk-headless screen
 
 # install minecraft server
-curl https://launcher.mojang.com/v1/objects/1b557e7b033b583cd9f66746b7a9ab1ec1673ced/server.jar -o server.jar
+# download minecraft server if not already downloaded
+[[ ! -e server.jar ]] \
```

Comment:
Could we just add an "if" here conditional?

Reply:
I have no idea but sounds like a thing

Reply:
It would make the `[[ stuff ]]` more explicit with `if [[ conditional statement ]]; then DO X; fi;`

Reply:
:+1:

Reply:
Thumbs Up.

Reply:
👍